lightstep/constants.go

```diff
@@ -11,7 +11,6 @@ var (
 		"40m": 2400000,
 		"50m": 3000000,
 		"1h":  3600000,
-		"1":   5400000,
```

Comment:
While here I noticed this very confusing value that is allowed for the interval. After discussing internally we decided to remove this for clarity.